components/Order/index.rst

```diff
@@ -4,10 +4,12 @@ Order
 E-Commerce PHP library for creating and managing sales orders.
 
 .. toctree::
-   :numbered:
+   :maxdepth: 2
 
    installation
-   order_and_items
-   adjustments
+   models
+   interfaces
    order_states
-   summary
+   order_events
+   order_transitions
+   basic_usage
```

Comment:
Exactly, the first thing you look for is `How to use it` and then if you need more details about some parts you look for them :wink: